src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
you didnt understand huh, the pool that uses nicknyan is different! it uses random event!! the iterator makes the colors are ordered like
1
2
3
4
5
6
7
8
...